release-notes.txt

```diff
@@ -1,3 +1,30 @@
+OpenMDAO Version 1.7.0 Alpha Release Notes
+June 30, 2016
+
+Backwards-Incompatible Change:
+* Change fd_options to be more self consistent, changed name to deriv_options.
+
+Features:
+* NAS access component
+* Derivatives output adds an asterisk to the 'Absolute Error' and 'Relative Error'
+  lines if those lines show a violation greater than some threshold value (1.0E-6 by default).
+* User can now specify scaler and resid_scaler in add_output and add_state metadata.
+* A host of improvements to the model structure viewer.
```

Comment:
Actually, we never announced it before, did we? So -- A new Model Structure Viewer has been added for viewing the data dependency of your model. (Justin may have some more exciting wording for this.)